src/main/resources/db/migration/V1__CREATE_TABLES.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE SUBSUMSJON (
```

Comment:
It may be safer to do `CREATE TABLE IF NOT EXISTS SUBSUMSJON (..` ?

Reply:
We also may need a timestamp? 

`  timestamp      TIMESTAMP WITH TIME ZONE NOT NULL default (now() at time zone 'utc'),`